src/features/auth/register-form/RegisterForm.tsx

```diff
@@ -16,65 +29,102 @@ const RegisterForm: FunctionComponent<props> = ({ disabled, onSubmit }) => {
-        onChange={onChangeEmail}
-        disabled={disabled}
-      />
-      <TextField
-        value={password}
-        label="Contraseña"
-        type="password"
-        onChange={onChangePassword}
-        disabled={disabled}
-      />
-      <TextField
-        value={confirmPassword}
-        label="Confirmar contraseña"
-        type="password"
-        onChange={onChangeConfirmPassword}
-        disabled={disabled}
-      />
-      <Button
-        variant="contained"
-        color="primary"
-        type="submit"
-        disabled={disabled}
-        onClick={() => {
-          onSubmit(name, lastname, email, password, confirmPassword);
-        }}
-      >
-        Registrarse
-      </Button>
+      <Box sx={FormBoxStyles}>
+        <Box sx={ImageBoxStyles}>
+          <img src={ucabLogo} alt="UCAB Logo" style={UcabLogoStyles} />
+        </Box>
+        <Typography sx={TextFieldTypography}>Nombre</Typography>
+        <TextField
+          value={name}
+          label=""
+          type="name"
+          onChange={onChangeName}
+          disabled={disabled}
+          sx={TextFieldStyles}
+          InputProps={{ sx: { borderRadius: LOGIN_BORDER_RADIUS } }}
+        />
+        <Typography sx={TextFieldTypography}>Apellido</Typography>
+        <TextField
+          value={lastname}
+          label=""
+          type="lastname"
+          onChange={onChangeLastname}
+          disabled={disabled}
+          sx={TextFieldStyles}
+          InputProps={{ sx: { borderRadius: LOGIN_BORDER_RADIUS } }}
+        />
+        <Typography sx={TextFieldTypography}>Correo electronico</Typography>
+        <TextField
+          value={email}
+          label=""
+          type="email"
+          onChange={onChangeEmail}
+          disabled={disabled}
+          sx={TextFieldStyles}
+          InputProps={{ sx: { borderRadius: LOGIN_BORDER_RADIUS } }}
+        />
+        <Typography sx={TextFieldTypography}>Contraseña</Typography>
+        <TextField
+          value={password}
+          label=""
+          type="password"
+          onChange={onChangePassword}
+          disabled={disabled}
+          sx={TextFieldStyles}
+          InputProps={{ sx: { borderRadius: LOGIN_BORDER_RADIUS } }}
+        />
+        <Typography sx={TextFieldTypography}>Confirmar contraseña</Typography>
+        <TextField
+          value={confirmPassword}
+          label=""
+          type="password"
+          onChange={onChangeConfirmPassword}
+          disabled={disabled}
+          sx={{ ...TextFieldStyles, marginBottom: "26px" }}
+          InputProps={{ sx: { borderRadius: LOGIN_BORDER_RADIUS } }}
+        />
+        <Button
+          variant="contained"
+          color="primary"
+          type="submit"
+          disabled={disabled}
+          onClick={() => {
+            onSubmit(name, lastname, email, password, confirmPassword);
+          }}
+          sx={ButtonStyles}
+        >
+          Registrarse
+        </Button>
```

Comment:
Agregar Link al Inicio de sesion
```suggestion
        <Link to:"/register"> Ya tienes una cuenta? Inicia Sesion</Link>
```